.github/workflows/ci.yml

```
@@ -0,0 +1,53 @@
+name: Test Branch
```

```suggestion
name: Run tests
```
WDYT? Test branch seems a little bit off to me, after the recent changes